national_rail/requirements.txt

```diff
@@ -1,3 +1,5 @@
 requests
 python-dotenv
-pytest
\ No newline at end of file
+pytest
+beautifulsoup4
```

Comment:
Should this be `bs4` as you imported `bs4`?

Reply:
Pip freeze gives me beautifulsoup4